README.md

```diff
@@ -111,7 +111,7 @@ Note: append mode is the default behavior when no `regex` match is found for bac
                 - name: Checkout
                   uses: actions/checkout@v3
```

Comment:
Can we also update this to the latest to match the above examples?
```suggestion
                  uses: actions/checkout@v4
```

Reply:
Done 💪